tests/ast/data/DataFrame.join.inner.predicate_rsuffix.test

```diff
@@ -0,0 +1,17 @@
+## TEST CASE
+
+df1 = session.table("test_table")
+
+df2 = session.table("test_table")
+
+df3 = df1.join(df2, (df1["num"] < df2["num"]) & (df1["str"] == df2["str"]), rsuffix="_2").select('*')
+
+## EXPECTED OUTPUT
+
+res1 = session.table("test_table")
+
+res2 = session.table("test_table")
+
+res3 = res1.join(res2, how=(res1["num"] < res2["num"] & res1["str"] == res2["str"]), on="inner")
```

Comment:
I think this is wrong -- `how` and `on` are switched?

Reply:
Good catch, thank you! This was a bug in the unparser. Updated the unparser changes and the expectation tests.